.travis.yml

```diff
@@ -3,24 +3,18 @@ language: python
 services:
   - docker
 before_install:
-  # Prime the cache. We currently manually keep this synced.
-  - docker pull quay.io/openai/gym:test
-  # - docker build -f test.dockerfile -t quay.io/openai/gym:test .
-  # ^^^ commented out by joschu 2017/9/3
-  # build command fails due to apt issue "The following packages have unmet dependencies:"
+  - docker build -f test.dockerfile -t gym-test --build-arg=MUJOCO_KEY=$MUJOCO_KEY .
 install: "" # so travis doesn't do pip install requirements.txt
 script:
-  # In a pull request, there are no secrets, and hence no MuJoCo:
-  # https://docs.travis-ci.com/user/pull-requests#Security-Restrictions-when-testing-Pull-Requests.
-  - docker run -e MUJOCO_KEY_BUNDLE="${MUJOCO_KEY_BUNDLE:-}" quay.io/openai/gym:test tox
+  - docker run gym-test tox
 
 notifications:
   slack:
     secure: h/Mxm8K+avH/2W0818zCHmLloRPMFN4NJL01+VShvAkH80/acfjeq/+mMdWXXPL/oOB6kSHDk+GDhwR6+s03ZcPMn5INTFvFYqUc6UWmT+NXtOPxGTN0xda6MdYUkWQUKaMyjFrweZQOMOASFBIzPOq4XeVbM5aB8s4EJhnfAcYZhp/idwKbToVihN4KZgxlvZIFc8iEp1o9uSl5qrsaeYYYXRkb6mauacAwOo4/Chu+cOnoLUOnvhBFE3rV3doDNrbnoalO8XiExtgx5CIAYWrlMni7r2Q+LlzgwdyTH19ZtybPxJTZIIWSBQ2UtcoYdIEDcc36GcUwz1VUGg32mLJJnY2xw80CWR4ixFPpLwwP5Y99WTn8v094B4nmFTWOwNWXp3EkqtTN9XcJoRBqXB5ArucIPqrx57dOCljSKx22gL6WaF2p3stSAxIGFektGyGnisaELrFZG1C63aHoUPicj3gUlijmAoUmYaDRf6P1wnpXqBpKDAWWhAMSatvx1ekmEJgR7OQklQnnfjx9kENDUygNUWS4IQwN2qYieuzHFL3of7/30mTM43+Vt/vWN8GI7j01BXu6FNGGloHxjH1pt3bLP/+uj5BJsT2HWF+Z8XR4VE6cyVuKsQAFgCXwOkoDHALbcwsspONDIt/9ixkesgh1oFt4CzU3UuU5wYs=
     on_success: change
-  webhooks:
-    urls:
-      - https://hooks.zapier.com/hooks/catch/1711022/6ztmzh/
-      - https://hooks.zapier.com/hooks/catch/1711022/6zhc8p/
-    on_success: always
-    on_failure: always
+  # webhooks:
```

Comment:
so what exactly do these webhooks do? When curling them, I get some generic JSON success message (`{"status": "success", "attempt": "5af229fc-b676-4f1e-8436-1886aa2a3a10", "id": "4d7cd20f-76e2-47c4-a2ee-f742a35a0267", "request_id": "5af229fc-b676-4f1e-8436-1886aa2a3a10"}` 
Commenting out to see if anyone would cry of their absense :)

Reply:
Legacy, no issue with removing!

Reply:
okay, will remove